features/guidance/rotary.feature

```diff
@@ -90,19 +90,19 @@ Feature: Rotary
             | bcegb  | roundabout |
 
        When I route I should get
-           | waypoints | route       | turns                      |
-           | b,d       | bcegb,cd,cd | depart,bcegb-exit-1,arrive |
-           | b,f       | bcegb,ef,ef | depart,bcegb-exit-2,arrive |
-           | b,h       | bcegb,gh,gh | depart,bcegb-exit-3,arrive |
-           | c,f       | bcegb,ef,ef | depart,bcegb-exit-1,arrive |
-           | c,h       | bcegb,gh,gh | depart,bcegb-exit-2,arrive |
-           | c,a       | bcegb,ab,ab | depart,bcegb-exit-3,arrive |
-           | e,h       | bcegb,gh,gh | depart,bcegb-exit-1,arrive |
-           | e,a       | bcegb,ab,ab | depart,bcegb-exit-2,arrive |
-           | e,d       | bcegb,cd,cd | depart,bcegb-exit-3,arrive |
-           | g,a       | bcegb,ab,ab | depart,bcegb-exit-1,arrive |
-           | g,d       | bcegb,cd,cd | depart,bcegb-exit-2,arrive |
-           | g,f       | bcegb,ef,ef | depart,bcegb-exit-3,arrive |
+           | waypoints | route       | turns                           |
+           | b,d       | bcegb,cd,cd | depart,exit rotary right,arrive |
+           | b,f       | bcegb,ef,ef | depart,exit rotary right,arrive |
+           | b,h       | bcegb,gh,gh | depart,exit rotary right,arrive |
```

Comment:
I think so, it makes no sense to start on roundabout, but it is possible to make a re-routing request being inside a roundabout, so information about exit number will be very helpful.